pom.xml

```diff
@@ -271,45 +268,49 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>com.spotify</groupId>
-                        <artifactId>dockerfile-maven-plugin</artifactId>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>tag-version</id>
+                                <id>docker-build</id>
+                                <phase>install</phase>
                                 <goals>
                                     <goal>build</goal>
-                                    <goal>push</goal>
                                 </goals>
-                                <configuration>
-                                    <tag>${dockerfile.versionTagName}</tag>
-                                </configuration>
                             </execution>
                             <execution>
-                                <id>tag-latest</id>
+                                <id>docker-push</id>
+                                <phase>deploy</phase>
                                 <goals>
-                                    <goal>tag</goal>
                                     <goal>push</goal>
                                 </goals>
-                                <configuration>
-                                    <tag>${dockerfile.latestTagName}</tag>
-                                </configuration>
                             </execution>
                         </executions>
                         <configuration>
                             <skip>true</skip>
-                            <!-- Dockerfile plugin requires <repository> to be non-empty even if <skip> is true -->
-                            <repository>${dockerfile.repository}</repository>
-                            <buildArgs combine.children="append">
```

Comment:
This line was there to make it easy to pass more properties into the dockerfile from the child POM.
How is this achieved in the new configuration?

Reply:
It's only needed for adding elements of the same name. Since args have each different name there is no need for it.